diff --git a/index.d.ts b/index.d.ts
index 3e410b8..3dccd77 100644
--- a/index.d.ts
+++ b/index.d.ts
@@ -1,9 +1 @@
-// Type definitions for left-pad 1.2.0
-// Project: https://github.com/stevemao/left-pad
-// Definitions by: Zlatko Andonovski, Andrew Yang, Chandler Fang and Zac Xu
-
-declare function leftPad(str: string|number, len: number, ch?: string|number): string;
-
-declare namespace leftPad { }
-
-export = leftPad;
+// Oh right, this shouldn't exist either
